core/src/commonMain/kotlin/io/github/aag/core/data/network/models/AdditionalMaterialDTO.kt

```diff
@@ -14,3 +14,11 @@ data class AdditionalMaterialDTO(
 
 fun AdditionalMaterialDTO.toAdditionalMaterial(): AdditionalMaterial =
     AdditionalMaterial(topicName = topicName, url = url)
+
+fun fromAdditionalMaterial(additionalMaterial: AdditionalMaterial): AdditionalMaterialDTO =
```

Comment:
Can be extension function

Reply:
The same here, I don't want to open up data layer internals